### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
Updated README:
- Naming change from `InAppMessaging` to `RInAppMessaging`
- Added description for `accessibilityCompatibleDisplay` flag

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ pod install
 
 | Key     | Value     |
 | :---:   | :---:     |
-| `InAppMessagingSubscriptionID` | your_subscription_key |
+| `InAppMessagingAppSubscriptionID` | your_subscription_key |
```
Comment (donnie-jp) [inline README.md:34]: Good catch ⚾

File: README.md [hunk 2]
```diff
@@ -4,17 +4,17 @@
 
 # RInAppMessaging
 
-InAppMessaging allows app developers to easily configure and display notifications within their app.
+RInAppMessaging allows app developers to easily configure and display notifications within their app.
```
Comment (donnie-jp) [inline README.md:7]: When we're not referring to a CocoaPod name or a Swift class/object lets use IAM except here where we should introduce it as `In-App Messaging (IAM) module`.

File: README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ This method is provided for the host application to log and save events. These e
 func logEvent(_ event: Event)
 ```
 
-InAppMessaging provides three pre-defined event types and a custom event type:
+RInAppMessaging provides three pre-defined event types and a custom event type:
```
Comment (donnie-jp) [inline README.md:81]: `IAM`

File: README.md [hunk 4]
```diff
@@ -132,12 +132,12 @@ CustomAttribute* timeAttribute = [[CustomAttribute alloc] initWithKeyName:@"time
...
 NSArray* attriList = @[stringAttribute, intAttribute, boolAttribute, doubleAttribute, timeAttribute];
 CustomEvent* customEvent = [[CustomEvent alloc] initWithName:@"any_event_name_here" withCustomAttributes: attriList];
-[InAppMessaging logEvent:customEvent];
+[RInAppMessaging logEvent:customEvent];
 ```
 
 ### **registerPreference()**
 
-A preference is what will allow InAppMessaging to identify users for targeting and segmentation. At the moment, IAM will take in any of the following identifiers:
+A preference is what will allow RInAppMessaging to identify users for targeting and segmentation. At the moment, IAM will take in any of the following identifiers:
```
Comment (donnie-jp) [inline README.md:140]: `IAM`

File: README.md [hunk 5]
```diff
@@ -155,12 +155,12 @@ IAMPreference* preference = [[[[[[IAMPreferenceBuilder alloc] init]
...
     build];
 
-[InAppMessaging registerPreference:preference];
+[RInAppMessaging registerPreference:preference];
 ```
 
 ## **Custom Events**
 
-As shown in the example above for event logging, InAppMessaging also supports custom event. Custom events are events with an unique name and a list of attributes associated with them.
+As shown in the example above for event logging, RInAppMessaging also supports custom event. Custom events are events with an unique name and a list of attributes associated with them.
```
Comment (donnie-jp) [inline README.md:163]: `IAM`

File: README.md [hunk 6]
```diff
@@ -177,3 +177,34 @@ From the dashboard side, you will have the ability to also add an operator. The
 *Note:* When comparing date as timeInMillis values, there will be a tolerance of 1000 milliseconds. This means that comparisons using any relevant operator types will have a leniency of 1 second e.g. comparing 300ms and 600ms with the `EQUALS` operator will return `true` and comparing 300 and 1400 will return `false`.
 
 From the SDK side, host app developers will be able to log custom events as shown in the examples above. When the event matching process happens, note that the attributes of the event logged by the host app will be compared against the campaign's trigger attribute value e.g. if the trigger attribute value is an integer of 5 with an operator type of `GREATER_THAN`, and the attribute value of the event logged is an integer 10, then the 10 will be successfully matched against the 5 with a `GREATER_THAN` operator (i.e. 10 > 5).
+
+
+## **Optional features**
+
+### **(BOOL)accessibilityCompatibleDisplay**  
+
+This flag can be used to support UI test automation tools like Appium. When set to `true`, SDK will use different display method changing campaign messages' view hierarchy. This can solve issues with accessibility tools having problems detecting visible items.
```
Comment (donnie-jp) [inline README.md:186]: "the SDK will use a different display method which changes campaign messages' view hierarchy" is maybe better wording?

Comment (imaginaris) [reply README.md:186]: Yes. Much better :)

**Pull request merged**